t/theeser_tokenizer/theseer_tokenizer_1.2.0_1.2.1_rhel_8.4.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : theseer/tokenizer
+# Version       : 1.2.0, 1.2.1
```

Comment:
As per line 18, build script is created for version 1.2.0. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.